### PR TITLE
issue tracker: Support explicit label mappings for substituted labels.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR

File: src/appengine/libs/issue_management/issue_tracker_policy.py [hunk 1]
```diff
@@ -93,21 +113,27 @@ def get_new_issue_properties(self, is_security, is_crash):
...
+    labels = issue_type.get('labels')
+    if labels:
+      policy.labels.extend(_to_str_list(labels))
 
     if is_crash:
-      if 'crash_labels' in issue_type:
-        policy.labels.extend(issue_type['crash_labels'])
+      crash_labels = issue_type.get('crash_labels')
+      if crash_labels:
+        policy.labels.extend(_to_str_list(crash_labels))
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/issue_tracker_policy.py]: is this only an issue when you do labels.extend, how about labels.add ? we could also overwrite the add and extend in LabelStore itself if we are forcing labels to be str ?

Comment (oliverchang) [reply src/appengine/libs/issue_management/issue_tracker_policy.py]: Nice suggestion! I'll made add/remove in LabelStore force the arguments to be str, but in this case |labels| isn't a LabelStore, but a regular list, so these are still needed.

**Pull request merged**